### PR TITLE
Fix compatibility with PHP 8.1

### PR DESCRIPTION
**Pull request by janlanger**
These changes were made in master - https://github.com/webonyx/graphql-php/commit/451834341593bef6b82d080e2428e19edd59c4f5#diff-5f4e849a2f284951ac6488a3b3b386a9b0778daed44f61397dd89e398e3c53af but not ported to 14.x

### CODE REVIEW

**Comment by spawnia** [comment]
I am up for backporting this change, given CI passes.

**Comment by janlanger** [comment]
@spawnia CS and phpstan fail should be fixed

File: src/Utils/MixedStore.php [hunk 1]
```diff
@@ -184,7 +186,7 @@ public function offsetGet($offset)
      *
      * @return void
```
Comment (ruudk) [inline src/Utils/MixedStore.php:187]:
```suggestion
```

File: src/Utils/MixedStore.php [hunk 2]
```diff
@@ -220,7 +222,7 @@ public function offsetSet($offset, $value)
...
```
Comment (ruudk) [inline src/Utils/MixedStore.php:223]:
```suggestion
```

**Review by ruudk — COMMENTED** [review]
To fixes needed to make the PR green.

**Pull request merged**